### ISSUE TITLE
Brighter Apex Somber + Live Another Life Starts

### ISSUE THREAD

**Issue by SovnSkyrim** [jdsmith2816/eldersouls#194]
First, whenever you see this, i can send you a brighten up Apex Somber. Some examples of Interiors :
Cities
http://prntscr.com/w9rw3o
http://prntscr.com/w9rzgs
http://prntscr.com/w9s034
http://prntscr.com/w9s0oj
http://prntscr.com/w9s15m
Bleak Falls Barrow
http://prntscr.com/w9rx6e
http://prntscr.com/w9rxom
http://prntscr.com/w9rwrt
http://prntscr.com/w9ryo1
http://prntscr.com/w9rz0g

I can even do with a bit more brightness but I feel it loses a bit of it's dark feeling http://prntscr.com/w9s2hv (compare this with http://prntscr.com/w9s15m )

Second; yesterday and today I tried almost all the starts, here's the feedback: 

- I arrived on board of a ship. : Should be alright since it's all cities and you can survive easily, even with the cold (due to carriages)
- I own property in one of the holds. : I personally don't like it as it's absurdly OP (having a house right away) and may conflict with Vanilla house replacers.
- I'm a new member of a guild. : All the options are feasible, being able to start the DLC right away isn't recommended tho (start as dawnguard/vampire) but not impossible. It's especially rng dependent as the radiant quests for the companions can send the player to anything (falmer or forgotten dungeons) which can be ultra hard. Same for the college with Saarthal, thieves guild/Dark brotherhood are easier.
- I'm a patron at a local inn. : One of the hardest as you literally have nothing but some gold and an iron sword. Although not inherently impossible because of spawns, I had a hard time doing anything but missives when trying it.
- I'm a soldier in the army. : The jagged crown is very hard. Draugrs this early is a dealbreaker, especially a buffed named one. It's possible to complete it as you have your crew of soldiers but it's one heck of a challenge (but less than the rpg dependent companion quests).
- I got caught crossing the border illegally. [Vanilla Start] : Totally alright, the game works just fine doing it, the only problem is a minotaur that is very easy to aggro right outside the prison exit.
- I want to escape this cell. : Honestly, this one is straight up impossible, the starting area has Mihail Enemies that one shot you (a wraith) and zombies that fuck you up if you succesfully baited the wraith, and that's on top of the normal enemies in there which were already hard (but not impossible) to deal with.
- I'm an outlaw in the wilds. : I personally hate this one as it puts the player in the bandit faction rendering all guards hostile. It's a very unique take and not inherently impossible, but I personally don't ever use it.
- I am a Vigilant of Stendarr. : My favourite start which is hardcore in this list. A lot (and I mean A LOT) of mihail placed enemies are around, and reaching Dawnstar to take a carriage is a little challenge but not impossible. I expect it to be way harder with The Frozen North as I have no clue how Vigilant Gear scales in warmth.
- I am camping in the woods. : As easy but with less gear helgen start. Same commentary: Care about the Minotaur or it's gonna behead you.
- I was shipwrecked off the coast! : One of the hardest with a hypothermia mod as you get barely anything to reach a city to sell all that junk to make a misery and buy some gear. The area has a bunch of mihail monsters that may also be very frustrating.
- I am a vampire in a secluded lair. : Fine start, except if the player starts to attack the vampires he starts with. Some locations have big mihail monsters right outside tho (Pinemoon cave for example) so that may be annoying.
- I am a necromancer in a secret location. : I'd be totally fine with it if one of the locations wasn't Sinderion's Laboratory in BLACKREACH. Seriously, it's impossible to get out of there, even if there was no Mihail stuff.
- I am a warlock's thrall. : Some locations may have annoying Mihail stuff around, but besides that, you start with basically nothing whatsoever so it's ultra hard (plus you cant even kill the necromancer that "enthralled you" because of all the Mihail enemies around.).
- I was attacked and left for dead! : One of the starts where you get attacked very easily as you spawn. Not impossible but I personally spawned in Solstheim multiple times and that wasn't very nice of you game!

**Comment by SovnSkyrim** [comment]
Feedback on New Beginnings - Live Another Life Extension.

- I am an addict in a skooma den. : Remote location, easy to get out but doing the redwater quest is dumb as that's dlc. Good start.
- I am an adventurer in a dwemer ruin. : Honestly the hardest start possible, in one of my attempts, I spawned right next to a lvl 44 Falmer Warmonger. This ons is should be removed if you add New Beginnings.
- I am an alchemist camping in the wilderness. : Very nice spot for an alchemist/mage start. No real enemies around the spawns. A nice addition.
- I have arrived at Angi's Camp in Falkreath hold. : Archery Start! You spawn near Angi so in the mountains, but the starting gear should be good enough for a hypothermia mod. Cool start overall.
- I am a beggar. : For the trully hardcore player. You spawn with nothing. Honestly unique.
- I was attacked by a dragon. : Another start that is hard due to the lack of gear. Still quite nice, starting in a burned down house, next to incinerated corpses, sets the tone.
- I have been thrown in jail. : A JAIL START LIKE IN Dark Souls 1! Amazing right? And the player starts with some gear! Even better! Good start overhaul.
- I am a wandering mercenary traveling Skyrim. : Quite the OP start as you have steel plate armor right away. But you can spawn in very cold areas which may be hard with a Hypothermia mod. Good for roleplay but op.
- I am a wandering merchant traveling Skyrim. : Even more op, you get a merchant iventory so you have a bit of everything and quite some gold. you also have a horse! Roleplay start but OP, same locations as wandering Mercenary too.
- I am a pilgrim to a temple. : Another "i'm poor" start but for mages. Not integrated with Wintersun so quite useless if you ask me. Roleplay Start.
- I was imprisoned by the Thalmor. : Prison start but hardcore. It's honestly possible but ultra hard, plus it's northwatch keep so you're in the north, cold af land.
- I was slain by a vampire and laid to rest. : Vampire start but not in the wilderness so you can try to do stuff! Yeaye! You're kinda naked when starting but should be possible to survive with vampire powers.
- I am a warlock in a secluded lair. : Very disturbing start due to all of the Mihail stuff around but they're all nice to you so you're able to live. Good Mage start.
- I am a werewolf at Frostmoon Crag. : A unique Werewolf Start on Solstheim. Honestly cool start.
- I am a pilgrim to a sacred site. : better gear than pilgrim to a temple but honestly just as boring. Some you die in the loading screen due to mihail monsters. Roleplay Start.


TL;DR Some are good, some are unique, some are dumb and impossible.

Placed right after LAL in LO with this patch 
[New Beginnings - Elder Souls.zip](https://github.com/jdsmith2816/eldersouls/files/5743598/New.Beginnings.-.Elder.Souls.zip)
which solves like 2/3 records to be consistent with the rest of the list.
Had 0 problems when play-testing it.
